### PR TITLE
Update package-lock.json files

### PR DESCRIPTION
**Pull request by JasonStoltz**
Lerna did NOT update package-lock.json files. I updated them all
and noted that during publishing you'll need to manually update
them.

### CODE REVIEW

File: ADVANCED.md [hunk 1]
```diff
@@ -951,6 +951,8 @@ Publish new version
 2. Run `npx lerna changed` to see which projects will be published.
 3. Update `CHANGELOG` files to include version `v0.6` for the projects that will be published.
 4. Run `npx lerna version 0.6 --exact`.
+   NOTE: Lerna does NOT update `package-lock.json` file, so at this point you'll likely have
+   to manually update them.
```
Comment (omegahm) [inline ADVANCED.md]: "[...] by running `yarn`"?

Comment (JasonStoltz) [reply ADVANCED.md]: > NOTE: Lerna does NOT update `package-lock.json` file, so at this point you'll have
   to manually edit the `package-lock.json` for each updated package to update
   `0.5` to `0.6` at the top of the file.

File: ADVANCED.md [hunk 2]
```diff
@@ -949,14 +949,14 @@ Publish new version
 
 1. Create new version branch `git checkout -b v0.6`.
```
Comment (omegahm) [inline ADVANCED.md]: Does this line also have to be `v0.6.0` to follow the rest of the version updates?

Comment (JasonStoltz) [reply ADVANCED.md]: That is actually intentionally "v0.6". That branch will track the latest version of v0.6, which will be a branch that is maintained in parallel to master.

**Pull request merged**